decisionengine/dijkstratesting/time_test.py

```diff
@@ -0,0 +1,364 @@
+                   'wLeft': [('oLeft', 6), ('uLeft', 1), ('zLeft', 1)],
+                   
+                   'xLeft': [('vLeft', 1), ('zLeft', 1)],
+                   
+                   'zLeft': [('xLeft', 1), ('wLeft', 1)],
+
+                   'aRight': [('a', 2),('bRight', 1), ('cRight', 3)],
+
+                   'bRight': [('aRight', 1), ('cRight', 3), ('eRight', 2), ('iRight', 1)],
+
+                   'cRight': [('aRight', 3), ('bRight', 1), ('dRight', 2)],
+
+                   'dRight': [('cRight', 2), ('eRight', 1), ('fRight', 2), ('gRight', 2),('hRight',1)],
+
+                   'eRight': [('dRight', 1), ('bRight', 2), ('hRight', 1)],
+
+                   'fRight': [('dRight', 2), ('gRight', 3), ('pRight', 3)],
+
+                   'gRight': [('dRight', 2), ('eRight', 1), ('pRight', 2), ('hRight', 2)],
+
+                   'hRight': [('eRight', 1), ('dRight', 2), ('lRight', 2)],
+                   
+                   'iRight': [('bRight', 1), ('jRight', 1), ('kRight', 3)],
+                   
+                   'jRight': [('iRight', 1), ('lRight', 1)],
+                   
+                   'kRight': [('iRight', 3), ('mRight', 1)],
+                   
+                   'lRight': [('hRight', 2), ('jRight', 1), ('sRight', 1)],
+                   
+                   'mRight': [('kRight', 1), ('nRight', 1)],
+                   
+                   'nRight': [('mRight', 1), ('oRight', 2)],
+                   
+                   'oRight': [('nRight', 2), ('wRight', 6)],
+                   
+                   'pRight': [('fRight', 3), ('gRight', 1), ('qRight', 1)],
+                   
+                   'qRight': [('rRight', 2), ('pRight', 1)],
+                   
+                   'rRight': [('qRight', 1), ('vRight', 2)],
+                   
+                   'sRight': [('lRight', 1), ('tRight', 1)],
+                   
+                   'tRight': [('sRight', 1), ('vRight', 2), ('uRight', 1)],
+                   
+                   'uRight': [('tRight', 1), ('wRight', 1)],
+                   
+                   'vRight': [('tRight', 2), ('rRight', 2), ('xRight', 1)],
+                   
+                   'wRight': [('oRight', 6), ('uRight', 1), ('zRight', 1)],
+                   
+                   'xRight': [('vRight', 1), ('zRight', 1)],
+                   
+                   'zRight': [('xRight', 1), ('wRight', 1)]
+                   }
+                   
+        self.g4 = {(1,1): [((1,2), 1)],
+                   (1,2): [((1,1),1), ((2,1), 1)],
+                   (2,1): [((1,2), 1)]
+                  }
+
+# Testing 1 path - graph with 26 nodes
+    def test_graph1(self):
+        tic = time.perf_counter()
+           
+        self.assertEqual(Dijkstras().dijkstras(self.g1, 'a', 'z'), (9, ['a', 'b', 'i', 'j', 'l', 's', 't', 'u', 'w', 'z']))
+        
+        toc = time.perf_counter()
+        
+        print(f"Graph 1 Dijkstras Test - 26 nodes {toc - tic:0.4f} seconds")
```

Comment:
Instead of an f-string, it looks cleaner to do a String.format().

Ex: `'This is a print statement saying {}'.format('Hello World!')`